From: https://gitlab.gnome.org/GNOME/gtk/-/merge_requests/10014
Subject: monitor/wayland: Don't leak zxdg_output_v1 proxy

diff --git a/gdk/wayland/gdkmonitor-wayland.c b/gdk/wayland/gdkmonitor-wayland.c
--- a/gdk/wayland/gdkmonitor-wayland.c
+++ b/gdk/wayland/gdkmonitor-wayland.c
@@ -38,6 +38,7 @@ gdk_wayland_monitor_finalize (GObject *object)
   g_free (monitor->name);
 
   wl_output_destroy (monitor->output);
+  g_clear_pointer (&monitor->xdg_output, zxdg_output_v1_destroy);
 
   G_OBJECT_CLASS (gdk_wayland_monitor_parent_class)->finalize (object);
 }
